src/data/markdown/docs/02 javascript api/06 k6-execution.md

```diff
@@ -74,6 +74,13 @@ Meta information and execution details about the current vu and iteration.
 | iterationInScenario | integer | The identifier of the iteration in the current scenario. |
 | idInInstance        | integer | The identifier of the VU across the instance. |
 | idInTest            | integer | The globally unique (across the whole test run) identifier of the VU. |
+| tags                | object  | The map that gives control over VU's Tags. The Tags will be included in every metric emitted by the VU and the Tags' state is maintained across different iterations of the same Scenario while the VU exists. |
+
+> ℹ️ Set Tag
+>
+> Set a Tag with the same key as a [SystemTag](docs/using-k6/options#system-tags) is allowed but it requires attention to avoid unexpected results. Overwriting SystemTags will not throw an error, but in most cases will not actually change the value of the emitted metrics as expected. For example, overwriting `url` will not result in a changed value when `http.get` is called as it will set its own value, but it will add the tag `url` to the metric emitted by a `check()` or `metric.add()`. This is not true for the `name` tag and `http.*` methods as that is already supported for the purposes of the [URL Grouping](https://k6.io/docs/using-k6/http-requests/#url-grouping) feature.
+>
+> Not all the types are accepted for a Set operation: it supports String, Number and Boolean types. Under the hood, the `tags` object handles a Tag as a String pair of key and value, so all the types will be implicitly converted into a String. If one of the denied types is used (e.g. Object or Array) and the `--throw` option is set then an exception is thrown otherwise just a warning is printed and the Set operation is discarded.
```

Comment:
```suggestion
> ℹ️ Setting tags
>
> Seting a Tag with the same key as a [system tag](docs/using-k6/options#system-tags) is allowed, but it requires attention to avoid unexpected results. Overwriting system tags will not throw an error, but in most cases will not actually change the value of the emitted metrics as expected. For example, trying to set the `url` tag value will not result in a changed tag value when `http.get()` is called, since the tag value is determined by the HTTP request itself. However, it will add the tag `url` to the metric samples emitted by a `check()` or `metric.add()`, which is probably not the desired behavior. On the other hand, setting the `name` tag will work as expected, since that was already supported for `http.*` methods, for the purposes of the [URL Grouping](https://k6.io/docs/using-k6/http-requests/#url-grouping) feature.
>
> Not all the types are accepted as a tag value: k6 supports strings, numbers and boolean types. Under the hood, the `tags` object handles a Tag as a `String` key-value pair, so all the types will be implicitly converted into a string. If one of the denied types is used (e.g. Object or Array) and the [`throw` option](https://k6.io/docs/using-k6/options/#throw) is set, an exception will be thrown. Otherwise, only a warning is printed and the tag value will be discarded.
```